pkg/autoscaler/collector_test.go

```diff
@@ -326,18 +324,26 @@ func TestMetricCollectorError(t *testing.T) {
 
 	for _, test := range testCases {
 		t.Run(test.name, func(t *testing.T) {
+			defer ClearAll()
+			logger := TestLogger(t)
 			factory := scraperFactory(test.scraper, nil)
 			coll := NewMetricCollector(factory, logger)
 			coll.CreateOrUpdate(test.metric)
+			key := types.NamespacedName{Namespace: test.metric.Namespace, Name: test.metric.Name}
 
 			var got duckv1beta1.Status
-			wait.PollImmediate(10*time.Millisecond, 2*time.Second, func() (bool, error) {
-				got = test.metric.Status.Status
-				return equality.Semantic.DeepEqual(got, test.expectedMetricStatus), nil
+			wait.PollImmediate(10*time.Millisecond, 5*time.Second, func() (bool, error) {
```

Comment:
why did we change time?

Reply:
Yeah this is not necessary. Just removed